hr_contract_currency/models/hr_contract.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrContract(models.Model):
+    _inherit = 'hr.contract'
+
+    currency_id = fields.Many2one(
```

Comment:
@pedrobaeza is this correct way of converting a related field to be non-related?

Reply:
+ another question: for some reason, changes on this field are not "tracked" in the bottom of the screen:
![image](https://user-images.githubusercontent.com/345102/47697913-d160b500-dc15-11e8-80c8-688f3798bccd.png)
a hint would be much appreciated

Reply:
Yes, although changing fields this way sometimes can be a problem with other modules. Let's hope it isn't in this case. For having that changes, you have to add `track_visibility` argument

Reply:
Awesome, thanks!